docs/reference/ml/anomaly-detection/apis/put-job.asciidoc

```diff
@@ -63,6 +63,9 @@ include::{es-repo-dir}/ml/ml-shared.asciidoc[tag=analysis-config]
 `bucket_span`:::
 (<<time-units,time units>>)
 include::{es-repo-dir}/ml/ml-shared.asciidoc[tag=bucket-span]
++
+deprecated:[8.1, "Setting `bucket_span` to a value that is neither an integral multiple
+nor divisor of a day is deprecated and will be disallowed in a future major version."]
```

Comment:
To me it was not obvious that a "divisor of a day" means "divisor of a number of seconds in a day".
Could it be stated more explicitly here and in other places?